src/west/log.py

```diff
@@ -40,9 +41,23 @@ def dbg(*args, level=VERBOSE_NORMAL):
     print(*args)
 
 
-def inf(*args):
-    '''Print an informational message.'''
+def inf(*args, colorize=False):
+    '''Print an informational message.
+
+    colorize (default: False):
+      If True, the message is printed in bright green if stdout is a terminal.
+    '''
+    # This approach colorizes any sep= and end= text too, as expected.
+    #
+    # The final flush=True avoids issues with unrelated output from commands
+    # (usually Git) becoming green, due to the final attribute reset ANSI
+    # escape getting line-buffered.
+    #
+    # colorama automatically strips the ANSI escapes when stdout isn't a
+    # terminal (by wrapping sys.stdout).
+    if colorize: print(colorama.Fore.LIGHTGREEN_EX, end='')
```

Comment:
Just a nit -- can you put the print() on its own line? flake8 complains about this.